app/helpers/react_components/mentoring/session.rb

```diff
@@ -8,6 +8,12 @@ def to_s
           "mentoring-session",
           {
             user_id: current_user.id,
+            relationship: SerializeMentorStudentRelationship.(student_mentor_relationship),
+            request: SerializeMentorSessionRequest.(request),
+            discussion: SerializeMentorSessionDiscussion.(discussion, current_user),
+            track: SerializeMentorSessionTrack.(track),
+            exercise: SerializeMentorSessionExercise.(exercise),
```

Comment:
It's nice having all of these properly serializing now I think. We can properly document them in the API docs too.

For Track and Exercise, we might want to reuse the full serializers for them eventually too. I couldn't make my mind up, so I've created the new "light" ones for now.